esti/presign_multipart_test.go

```diff
@@ -0,0 +1,246 @@
+package esti
```

Comment:
Can we add a test for u.size / u.partSize >= maxParts?

Reply:
:D sorry - I can manually test this one but it will be a big file to test